src/Orleans.Core/Configuration/OptionLogger/OptionFormatterExtensionMethods.cs

```diff
@@ -44,6 +55,20 @@ public static class OptionConfigureExtensionMethods
             return services;
         }
 
+        /// <summary>
+        /// Configure a option formatter for option <typeparam name="TOptions"/> if none is configured
+        /// </summary>
+        /// <typeparam name="TOptions"></typeparam>
+        /// <typeparam name="TOptionFormatter"></typeparam>
+        /// <param name="services"></param>
+        /// <returns></returns>
+        public static IServiceCollection TryConfigureFormatter<TOptions>(this IServiceCollection services)
```

Comment:
This function doesn't really do anything, as there will almost always be multiple IOptionFormatter in the system.  Should probably remove, as it will only confuse users.

Reply:
Are you talking about this specific Try method or both `TryConfigureFormatter` and `ConfigureFormatter`?

Reply:
Only this Try extension.  TryConfigureFormatter<TOptions>(..)

ConfigureFormatter<TOptions>(..) is fine

and the existing TryConfigureFormatter<TOptions<TFOptionsFormatter>
(..) is fine.